src/v2/System/Router/Boot.tsx

```diff
@@ -83,6 +84,7 @@ export const Boot = track(undefined, {
                     <ToastsProvider>
                       <Grid fluid maxWidth="100%">
                         <GlobalStyles />
+                        <OneTrustConsentStyles />
```

Comment:
I wonder how consent works. Is it something that we set on the user model? Or is it all 3rd party processed? Because if there's a place we check then this can be conditionally loaded via a dynamic import on a need-to basis.

Reply:
Right, which is why I'm thinking we should just copy out the compiled styles and not actually render this component in prod. It's partially geo, and then presumably there's a cookie (which I guess we could check that)

Reply:
I would assume _some_ need to change this stuff but not a lot of changes. Meaning I wouldn't want to "throw away" the code that generates this stylesheet even if that's what we end up with in production.